packages/template-retail-react-app/app/pages/product-detail/index.jsx

```diff
@@ -278,19 +278,6 @@ const ProductDetail = ({category, product, isLoading}) => {
 
                 {/* Product Recommendations */}
                 <Stack spacing={16}>
-                    <RecommendedProducts
-                        title={
-                            <FormattedMessage
-                                defaultMessage="Complete the Set"
```

Comment:
We'll need to make this removal conditional: only when there's a product set. Otherwise, we'd still want to show it.